index.html

```diff
@@ -40,5 +40,8 @@ <h2>Header Two</h2>
     <input type="number" id="phone" placeholder="Phone"><br>
```

Comment:
Kind check your input type for phone numbers. Look for the right input type to use.